test/integration/configuration-governance.v1.test.js

```diff
@@ -0,0 +1,1068 @@
+/* eslint-disable no-console */
+/**
+ * (C) Copyright IBM Corp. 2020.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+const ConfigurationGovernanceV1 = require('../../dist/configuration-governance/v1');
+const { readExternalSources } = require('ibm-cloud-sdk-core');
+const authHelper = require('../resources/auth-helper.js');
+const { v4: uuidv4 } = require('uuid');
+
+// testcase timeout value (25s).
+const timeout = 25000;
+
+// Location of our config file.
+const configFile = 'configuration_governance.env';
+
+const describe = authHelper.prepareTests(configFile);
+
+const verbose = true;
+let configurationGovernanceService;
+let configurationGovernanceServiceNoAccess;
+
+let TEST_LABEL;
+let ACCOUNT_ID;
+let TEST_SERVICE_NAME;
+let ENTERPRISE_SCOPE_ID;
+let SUBACCT_SCOPE_ID;
+
+// Generate a txn-id to be used during this test run.
+const transactionId = uuidv4();
+
+// Variables to hold various id's and object instances (these could perhaps be configured via links).
+let ruleId1;
+let rule1;
+let ruleEtag1;
+let ruleId2;
+
+let sampleRule1;
+let sampleRule2;
+let badSampleRule;
+
+let enterpriseScope;
+let accountScope;
+let badScope;
+
+let attachmentId1;
+let attachment1;
+let attachmentEtag1;
+let attachmentId2;
+
+describe('ConfigurationGovernanceV1_integration', () => {
+  jest.setTimeout(timeout);
+
+  beforeAll(async done => {
+    log('Starting setup...');
```

Comment:
I know I'm probably missing something because I haven't stayed too up to date with the generated integration tests, but where is `log` coming from?

Reply:
local function

Reply:
Wow, I just didn't `Ctrl-F` far enough 😅 

Thanks

Reply:
I won't tell anyone :)